plugins/status/plugin.go

```diff
@@ -454,3 +467,29 @@ func (p *Plugin) logUpdate(update *UpdateRequestV1) error {
 	}).Info("Status Log")
 	return nil
 }
+
+func updatePrometheusMetrics(ctx context.Context, u *UpdateRequestV1) {
+	pluginStatus.Reset()
```

Comment:
Is this safe to do? Since it's a method that hasn't been added here, I wonder if there could be undesired side effects 🔍

Reply:
pluginStatus is a variable (gauge vec) declared on the same package and it never will be nil. and it is concurrently safe (by mutex), the reset function just erases a map. I think it is safe, but if you have any concerns share with me ;)